src/platform/tests/Bridge/Mistral/Contract/DocumentNormalizerTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Platform\Tests\Bridge\Mistral\Contract;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Medium;
+use PHPUnit\Framework\TestCase;
+use Symfony\AI\Platform\Bridge\Mistral\Contract\DocumentNormalizer;
+use Symfony\AI\Platform\Bridge\Mistral\Mistral;
+use Symfony\AI\Platform\Contract;
+use Symfony\AI\Platform\Message\Content\Document;
+
+#[Medium]
+#[CoversClass(DocumentNormalizer::class)]
+final class DocumentNormalizerTest extends TestCase
+{
+    public function testSupportsNormalization()
+    {
+        $normalizer = new DocumentNormalizer();
+
+        $this->assertTrue($normalizer->supportsNormalization(new Document('some content', 'application/pdf'), context: [
+            Contract::CONTEXT_MODEL => new Mistral(),
+        ]));
+        $this->assertFalse($normalizer->supportsNormalization('not a document'));
+    }
+
+    public function testGetSupportedTypes()
+    {
+        $normalizer = new DocumentNormalizer();
+
+        $expected = [
+            Document::class => true,
+        ];
+
+        $this->assertSame($expected, $normalizer->getSupportedTypes(null));
+    }
+
+    #[DataProvider('normalizeDataProvider')]
+    public function testNormalize(Document $file, array $expected)
+    {
+        $normalizer = new DocumentNormalizer();
+
+        $normalized = $normalizer->normalize($file);
+
+        $this->assertEquals($expected, $normalized);
+    }
+
+    public static function normalizeDataProvider(): iterable
```

Comment:
Why using a data provider when we only have one entry. can you also add image and audio? or is it not supported?